src/5.0_to_5.1/__test_resources__/smallContentServer.ts

```diff
@@ -0,0 +1,1240 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.0";
+
+/**
+ * Content server object, useful for tests.
+ * Lighter version of `contentServer` in `./contentServer.ts`.
+ */
+export const smallContentServer: ContentRecord = {
```

Comment:
No actually you're right, let's rename it to `contentServer`.

Reply:
However, the risk I see is that with time, people enrich it with new content, to fit their need for new test use cases. And it might get bigger and not stay "small". Hence making clear now that this object is supposed to be the "small" content server might better.
Although as it concerns only the 5.0 migration, it might no be that much of a worry.
What do you think @antoinetissier?

Reply:
> However, the risk I see is that with time, people enrich it with new content, to fit their need for new test use cases.

If that's incremental like this then I think it's justified and not too much of an issue: you're going to add one specific dashboard to fit your test use case for example.

Even in the event where for some reason someone would try to merge a big structure on top of the `smallContentServer`, it should be caught by the review imo.

So personally I have a preference for `contentServer` for now, and only adapting later if needed.